docs/dashboards-classic/index.md

```diff
@@ -0,0 +1,34 @@
+---
+slug: /dashboards-classic
+title: Dashboards (Classic)
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/icons/dashboards.png')} alt="icon" width="50"/>
+
+:::caution
+We strongly recommend using our [Dashboards (New)](/docs/dashboards), which allows you to analyze metrics and log data on the same dashboard in a streamlined user experience. Sumo Logic will cease updates and support for Classic Dashboards.
```

Comment:
Do you need to remove "(New)" here?

Reply:
I have updated the sentence without this (New) word now.